ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java8/tasks/main.yml

```diff
@@ -4,26 +4,26 @@
 ##########
 - name: Test if Java 8 is already installed
   win_stat:
-    path: 'C:\Program Files\Java\jdk8u172-b11'
+    path: 'C:\openjdk\jdk8\bin'
```

Comment:
Whatever the name format, it should be consistent for 8,9,11; the others look more like this jdk-8

Reply:
All instances of `jdk8` have been changed to `jdk-8` and tested to make sure playbook deployment still works